openedx/features/ucsd_features/ecommerce/EcommerceClient.py

```diff
@@ -10,10 +10,34 @@
 
 
 class EcommerceRestAPIClient:
+    """
+    Client that communicates with the Ecommerce service
+    """
+
     def __init__(self, user, session=None):
+        """
+        Initialize the ecommerce client for the current user and session
+
+        Arguments:
+            user - User object
```

Comment:
```suggestion
            user: User object
```

Reference: https://edx.readthedocs.io/projects/edx-developer-guide/en/latest/style_guides/python-guidelines.html

Please change all of the docstrings according to edX standard.

Reply:
Done